src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -1696,4 +1723,1207 @@ export class PostgresCohortMembersService {
+          );
+        } else {
+          ShortlistingLogger.logShortlisting(
+            `No active forms found for cohort: ${cohort.cohortId}. Skipping shortlisting process for this cohort.`,
+            'ShortlistingEvaluation'
+          );
+          continue; // Skip this cohort entirely
+        }
+
+        // Check if any forms have valid rules structure and valid fieldIds
+        const hasValidRules = formFieldsAndRules.some(form => 
+          form.rules && form.rules.logic && form.rules.conditions
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)